### ISSUE TITLE
Support other projections

### ISSUE THREAD

**Issue by chelm** [ngoldman/bbox2extent#1]
It would be nice to be able to pass in projection/EPSG code instead of assuming 4326. I can make a PR to add but the question would be on what pattern would you want to follow 

```
bbox2extent(bbox, projCode, callback)
```

or 

```
bbox2extent(bbox, options, callback)
```

The 2nd would support potential future optional values

**Comment by ungoldman** [comment]
3rd option:

```
bbox2extent(bbox || options, callback)
```

First param could be bbox array or options object w/ bbox key/value inside, e.g.

```
bbox2extent({
  bbox: [0, 0, 0, 0],
  crs: {} // crs object http://geojson.org/geojson-spec.html#coordinate-reference-system-objects
}, callback)
```

...but yeah that's just down to how we would want to implement it. I agree it would be nice to support more projections.

**Comment by ungoldman** [comment]
http://proj4js.org/ would probably do the trick

**Comment by chelm** [comment]
yes proj4 would be the thing for sure to use. At the very least we could add the ability to specify an srid code `{srid: 102100}`. This would assume the data being passed in would be in some other known proj though. Not a big deal right now.

**Comment by ungoldman** [comment]
@chelm do you know much about the difference between specifying CRS and SRID? I don't fully understand the differences and implications.
